paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt

```diff
@@ -256,6 +263,27 @@ class Paparazzi(
     return TestName(packageName, className, methodName)
   }
 
+  private fun forcePlatformSdkVersion(targetSdkVersion: Int) {
+    val modifiersField: Field = Field::class.java
+        .getDeclaredField("modifiers")
+        .apply {
+          isAccessible = true
+        }
+
+    val versionClass = try {
+      Paparazzi::class.java.classLoader.loadClass("android.os.Build\$VERSION")
+    } catch (e: ClassNotFoundException) {
+      return
```

Comment:
Rethrow as `AssertionError`? This should never happen.

Reply:
happens in PaparazziTest